lib/puppet/type/sensu_api_config.rb

```diff
@@ -1,3 +1,14 @@
+# @!puppet.type.param name
```

Comment:
@ghoneycutt yes, but actually I was wondering if all the current types, and relevant providers, are going to remain as is or some of them are will be replaced by the works @jeffmccune is doing in https://github.com/sensu/sensu-puppet/pull/785 will affect some of the current ones, and their params. If so I wonder if it makes sense to prepare docs for types that are going to change soon.